src/coreclr/jit/llvm.cpp

```diff
@@ -1023,7 +1051,21 @@ void Llvm::fillPhis()
             unsigned       lclNum = phiArg->GetLclNum();
             unsigned       ssaNum = phiArg->GetSsaNum();
 
-            Value* localPhiArg = _localsMap->at({ lclNum, ssaNum });
+            Value* localPhiArg = nullptr;
+            auto iter = _localsMap->find({ lclNum, ssaNum });
+            if (iter == _localsMap->end())
+            {
+                //// Function args can be used as phi args without having seen them, and added them to _localsMap via a local.
+                //// But only for non shadow stack args with u:1
```

Comment:
```suggestion
                // Arguments are implicitly defined on entry to the method.
```
Don't think we need to be more elaborate than that.